### ISSUE TITLE
scrypt + opencl + multiple GPU

### ISSUE THREAD

**Issue by alphaonex86** [luke-jr/bfgminer#227]
Hello,

scrypt + opencl don't work only for one of my GPU juniper, but the GPU into the CPU (APU) seam work. I use bfgminer 3.0 64Bits.
With bitcoin work (no scrypt).
Linked with: https://github.com/luke-jr/bfgminer/issues/222
?

**Comment by alphaonex86** [comment]
--thread-concurrency 1 fix my problem

**Comment by alphaonex86** [comment]
cgminer work in this case.

**Comment by luke-jr** [comment]
So, practical effect is that BFGMiner getting a worse hashrate than cgminer?

**Comment by alphaonex86** [comment]
yes, with --thread-concurrency 1 have very worse hashrate, without it crash...

**Comment by alphaonex86** [comment]
--thread-concurrency 1 in reallity product hardware error, and never can mine.
